### PR TITLE
Update rule metadata

### PR DESCRIPTION
**Pull request by saberduck**
There were some EOL issues in some files, I corrected all of them, that's why there is "empty" diff in some files.

### CODE REVIEW

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4139.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "\"for in\" should not be used with iterables",
+  "title": "bar",
```
Comment (quentin-jaquier-sonarsource) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4139.json]: ???

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4787.html [hunk 2]
```diff
@@ -51,7 +51,7 @@ <h2>Recommended Secure Coding Practices</h2>
       <li> Cipher Block Chaining (CBC) with PKCS#5 padding (or PKCS#7) is susceptible to padding oracle attacks. </li>
     </ul> </li>
 </ul>
-<h2>Questionable Code Example</h2>
+<h2>Sensitive Code Example</h2>
```
Comment (nicolas-harraudeau-sonarsource) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4787.html]: I replaced the Questionable with Sensitive in the code example below

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4790.html [hunk 3]
```diff
@@ -34,7 +34,7 @@ <h2>Recommended Secure Coding Practices</h2>
   <li> the strength of hashing algorithms often decreases over time as hardware capabilities increase. Check regularly that the algorithms you are
   using are still considered secure. If needed, rehash your data using a stronger algorithm. </li>
 </ul>
-<h2>Questionable Code Example</h2>
+<h2>Sensitive Code Example</h2>
```
Comment (nicolas-harraudeau-sonarsource) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4790.html]: Same here. I replaced the Questionable with Sensitive in the code example below

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4817.html [hunk 4]
```diff
@@ -14,7 +14,7 @@ <h2>Ask Yourself Whether</h2>
 <p>You are at risk if you answered yes to this question.</p>
 <h2>Recommended Secure Coding Practices</h2>
 <p>Sanitize any user input before using it in an XPATH expression.</p>
-<h2>Questionable Code Example</h2>
+<h2>Sensitive Code Example</h2>
```
Comment (nicolas-harraudeau-sonarsource) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4817.html]: Same here. I replaced the Questionable with Sensitive in the code example below

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4818.html [hunk 5]
```diff
@@ -26,7 +26,7 @@ <h2>Recommended Secure Coding Practices</h2>
   <li> <a href="https://www.owasp.org/index.php/Input_Validation_Cheat_Sheet">Sanitize</a> any input read from the socket. </li>
   <li> Limit the number of sockets a given user can create. Close the sockets as soon as possible. </li>
 </ul>
-<h2>Questionable Code Example</h2>
+<h2>Sensitive Code Example</h2>
```
Comment (nicolas-harraudeau-sonarsource) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4818.html]: Same here. I replaced the Questionable with Sensitive in the code example below

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4823.html [hunk 6]
```diff
@@ -20,7 +20,7 @@ <h2>Recommended Secure Coding Practices</h2>
 <p>Any user or application can list running processes and see the command line arguments they were started with. There are safer ways of providing
 sensitive information to an application than exposing them in the command line. It is common to write them on the process' standard input, or give the
 path to a file containing the information.</p>
-<h2>Questionable Code Example</h2>
+<h2>Sensitive Code Example</h2>
```
Comment (nicolas-harraudeau-sonarsource) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4823.html]: Same here. I replaced the Questionable with Sensitive in the code example below

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4829.html [hunk 7]
```diff
@@ -13,7 +13,7 @@ <h2>Ask Yourself Whether</h2>
 <p>You are at risk if you answered yes to this question.</p>
 <h2>Recommended Secure Coding Practices</h2>
 <p><a href="https://www.owasp.org/index.php/Input_Validation_Cheat_Sheet">Sanitize</a> all data read from the standard input before using it.</p>
-<h2>Questionable Code Example</h2>
+<h2>Sensitive Code Example</h2>
```
Comment (nicolas-harraudeau-sonarsource) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4829.html]: Same here. I replaced the Questionable with Sensitive in the code example below

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S5122.html [hunk 8]
```diff
@@ -22,7 +22,7 @@ <h2>Recommended Secure Coding Practices</h2>
   <li> Use <code>Access-Control-Allow-Origin: *</code> only if your application absolutely requires it, for example in the case of an open/public API.
   For such endpoints, make sure that there is no sensitive content or information included in the response. </li>
 </ul>
-<h2>Questionable Code Example</h2>
+<h2>Sensitive Code Example</h2>
```
Comment (nicolas-harraudeau-sonarsource) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S5122.html]: Same here. I replaced the Questionable with Sensitive in the code example below

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4721.html [hunk 9]
```diff
@@ -35,7 +35,7 @@ <h2>Recommended Secure Coding Practices</h2>
   <li> refuse to run the command if the process has too many privileges. For example: forbid running the code as "root". </li>
 </ul>
 <p> </p>
-<h2>Questionable Code Example</h2>
+<h2>Sensitive Code Example</h2>
```
Comment (nicolas-harraudeau-sonarsource) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S4721.html]: I replaced the Questionable with Sensitive in the code example below

**Pull request merged**